src/styles/_media.scss

```diff
@@ -0,0 +1,20 @@
+@media (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+    width: 600px;
```

Comment:
why you hardcode width? Please check it
![image](https://user-images.githubusercontent.com/54936044/227708408-44cbcef9-03db-4aa9-ac54-07865a785ca0.png)